src/hotel-search/Merger.js

```diff
@@ -108,6 +121,52 @@ HotelSearchClient.prototype = {
     });
   },
 
+  _lastMergeRates: function(newRates) {
```

Comment:
could change to `_mergeRatesLastly` for better meaning ?

Reply:
I think it would be ok if we let it easy a little bit